### ISSUE TITLE
Cross-release for Scala 2.13

### ISSUE THREAD

**Issue by KarelCemus** [KarelCemus/play-redis#211]
This is blocked by:

- [x] Play framework release https://github.com/playframework/playframework/issues/9260
- [ ] Rediscala https://github.com/Ma27/rediscala/issues/21

**Comment by WayneWang12** [comment]
I don't understand. What's the difference between Ma27's fork and the origin https://github.com/etaty/rediscala. Seems etaty's redisscala has supported scala 2.13 several months ago.

**Comment by KarelCemus** [comment]
The difference is in a one bug fix. The issue was reported in https://github.com/etaty/rediscala/issues/160 and in this project in https://github.com/KarelCemus/play-redis/pull/151. I fixed this issue and created PR to Ma27's version since etaty's was dead for several months at that time. Therefore I switched versions. Recently, since Ma27 is not able to make a release for 2.13 and etaty's fork seemed slightly active, I also backported that PR as https://github.com/etaty/rediscala/pull/221 and it was merged. Unfortunately, there is no release ever since, which would include this bugfix.

In conclusion, there are two forks but neither of them addresses both: Scala 2.13 and the bugfix. Personally, I don't mind which fork I use, therefore I try to push both project to make a release so I could resolve this issue.

I hope I explained it sufficiently. Feel free to push the projects to make the release.

**Comment by WayneWang12** [comment]
@KarelCemus Yes you expained it quite clear. 🤣 Never thought it would be that interesting

**Comment by KarelCemus** [comment]
I forked the repo and released it myself

**Comment by KarelCemus** [comment]
Released as `2.5.0`
